src/ocm/hooks/useClusterSupportedPlatforms.ts

```diff
@@ -17,12 +17,13 @@ export default function useClusterSupportedPlatforms(clusterId: string) {
   });
 
   const isLoading = !error && !data;
+  const platformsSupported = ['vpshere', 'nutanix'];
   // Platform integration is supported
-  // if there is another platform type
-  // besides 'baremetal' or 'none', in the returned data.
+  // if there is a platform type
+  // inside platformssupported array, in the returned data.
   const isPlatformIntegrationSupported =
```

Comment:
Now it's not enough in `useClusterSupportedPLatforms` to return if there is an enabled platform integration, but it needs to indicate which it is.
So you should create a new field which is `supportedPlatformIntegration`, which can either be "undefined / vsphere / nutanix".